src/js/swarmAnimation.js

```diff
@@ -7,18 +7,28 @@
  * functions for applying css classes, doing transforms consistently
  */
 
-const swarmAnimation = {
-	hide(el) {
-		el.classList.add('anim-fade--out');
-	},
+const DURATION_TYPE = {
+	short: 'short',
+	medium: 'medium',
+	long: 'long'
+};
 
-	show(el) {
-		el.classList.remove('anim-fade--in');
+const swarmAnimation = {
```

Comment:
They are intended to be used with consumer apps, although these are intended more for non-React code. 

In React code, we would conditionally apply them in render and also using ReactCSSTransitionGroup's. This area is a little bit of a jump for me and I haven't tried it yet, although I could in this branch. And I'm not too sure when its better to just apply classes 
or use TransitionGroups. My initial understanding is that the TransitionGroup is used when things are added/removed enter/leave mounted/unmounted. I need to see if we have any components right now that have that behavior.
This may clear some things up:
https://medium.com/@joethedave/achieving-ui-animations-with-react-the-right-way-562fa8a91935